.babelrc

```diff
@@ -1,11 +1,20 @@
 {
 	"presets": [
-		"es2015",
+		[ "env", {
+			"targets": {
+				"browsers": [
+					"last 2 versions",
+					"Safari >= 9",
+					"iOS >= 9",
+					"not ie <= 10",
```

Comment:
No, for the same reason. `last 2 versions, ie > 10` includes `ie 10`. We could change `not ie <= 10` for `not ie 10` though.